src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -193,9 +193,11 @@ MulticopterAttitudeControl::generate_attitude_setpoint(const Quatf &q, float dt,
 	attitude_setpoint.timestamp = hrt_absolute_time();
 	_vehicle_attitude_setpoint_pub.publish(attitude_setpoint);
 
-	// update attitude controller setpoint immediately
-	_attitude_control.setAttitudeSetpoint(q_sp, attitude_setpoint.yaw_sp_move_rate);
```

Comment:
@MaEtUgR I guess it's done on purpose that in the MC attitude controller, when flying in Stabilized, the attitude setpoint that is generated here is directly sent to the attitude controller, and not passed over the `attitude_setpoint` uORB topic?